src/Codeception/Module/REST.php

```diff
@@ -1482,9 +1499,9 @@ public function grabAttributeFromXmlElement($cssOrXPath, $attribute)
      * @param $xml
      * @part xml
      */
-    public function seeXmlResponseEquals($xml)
+    public function seeXmlResponseEquals($xml): void
```

Comment:
$xml is string

Reply:
@Naktibalda I'm not sure about this... Couldn't it also be a `\DOMDocument` ?